sources/api/schnauzer/src/helpers.rs

```diff
@@ -907,6 +910,128 @@ pub fn goarch(
     Ok(())
 }
 
```

Comment:
For the helpers that you created, you will need to add them to the template registries that are used when templates are rendered. There are currently two codepaths that set up registries, and adding the functions to both is fine:
* [schnauzer-v1](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/8b0be519b7e3e64e1fa5a9eaa7873a6571d01565/sources/api/schnauzer/src/v1.rs#L111)
* [schnauzer-v2](https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/8b0be519b7e3e64e1fa5a9eaa7873a6571d01565/sources/api/schnauzer/src/v2/import/helpers.rs#L67)